templates/taxi/driver_list.html

```diff
@@ -6,8 +6,8 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.first_name }} {{ driver.last_name }}</a>
+          {% if user.username == driver.username %} (Me) {% endif %}
```

Comment:
Consider using the `pluralize` template filter for better code efficiency when displaying the number of drivers. This will help avoid conditional logic for singular and plural forms.